src/gui/widgets/TrackLabelButton.cpp

```diff
@@ -64,8 +64,17 @@ TrackLabelButton::TrackLabelButton( TrackView * _tv, QWidget * _parent ) :
 	}
 
 	setIconSize( QSize( 24, 24 ) );
-	setText( " " );
+	if( ConfigManager::inst()->value( "ui", "compacttrackbuttons" ).toInt() )
```

Comment:
@BaraMGB how well does compact buttons work on the constructor?  Previously it would fire a live-redraw but now I'd expect it to only take effect after the buttons are destroyed and recreated.  May be better suited for `update()` instead.